contracts/ENSRoot.sol

```diff
@@ -0,0 +1,99 @@
+pragma solidity ^0.4.18;
+
+import './ENS.sol';
+
+/**
+ * The ENS registry contract.
+ */
+contract ENSRoot is ENS {
```

Comment:
This should definitely be called 'ENSRegistry'; root implies it's only for the root node.